randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -3051,7 +3067,9 @@ Extra - asset_id: collision_camera_038_A
   * Extra - start_point_actor_name: SP_Checkpoint_EmmySancPhase2
   * Extra - start_point_actor_def: actordef:actors/logic/startpoint/charclasses/startpoint.bmsad
   > Tunnel to Purple EMMI Arena (Bottom)
-      After Ferenia - Wave Emmi Lower Water and Lay Bomb
+      After Ferenia - Wave Emmi Lower Water
+  > Tunnel to Purple EMMI Arena (Middle)
+      After Ferenia - Wave Emmi Lower Water
```

Comment:
This tunnel and the lower Purple EMMI Arena area are all that's left. This tunnel will need to require the EMMI being defeated.